### PR TITLE
Revision ontology tools

### PR DESCRIPTION
**Pull request by lucas-ubm**
Changes needed:
* Generated tables should have the same look as the manually created ones, we are missing horizontal lines which makes reading very uncomfortable for tools with long descriptions
* Some of the links in the generated tables are wrong (e.g. Cirgo). I fixed manually the ones I could, I think it happens when you query bio tools with complex names such as "https://bio.tools/circular%20gene%20ontology%20(cirgo)" instead of "https://bio.tools/cirgo"

### CODE REVIEW

**Comment by proccaserra** [comment]
@lucas-ubm thx for this. agree with the tables. interestingly, the tables in this recipe aren't mark down but HTML. The sister recipe (ETL) uses markdown tables. Don't know if this is due to the automated tool report or for another reason. Maybe @FuqiX and @EvaMart can comment on this . Thx everyone!

**Comment by FuqiX** [comment]
@EvaMart @proccaserra Please review the updated version.

Changes:
1. broken links replaced
2. HTML tables ->md tables
3. long description parsed

**Comment by EvaMart** [comment]
I checked all the links and they work. 
Two things I noticed:
1. In the description of "OntoText", there is a semicolon followed by a capitalized verb without an space between them. The semicolon should be replaced by a full stop and a space. 
2. Some descriptions have a full stop at the end and others do not. It would look better if all had it.

File: content/recipes/interoperability/ontology-operations-tools.md [hunk 1]
```diff
@@ -273,410 +55,138 @@ The table below is an overview of ontology strategies tools identified. Details
...
 |[OntoMaton](https://github.com/ISA-tools/OntoMaton)| A tool facilitating ontology search and tagging functionalities within Google Spreadsheets.|[CPAL license](https://opensource.org/licenses/CPAL-1.0)||Google Add-ons|
 |[Prodigy](https://prodi.gy/)|A modern annotation tool for creating training and evaluation data for machine learning models. You can also use Prodigy to help you inspect and clean your data, do error analysis and develop rule-based systems to use in combination with your statistical models.|Commercial license|Data annotation|Python, Web application,API|
 |[OntoText](https://www.ontotext.com/products/ontotext-platform/)|Connect and publish complex enterprise knowledge with standard-compliant semantic graph database;Customize and apply analytics to link documents to graphs, extract new facts, classify and recommend content...|Commercial license|
```
Comment (EvaMart) [inline content/recipes/interoperability/ontology-operations-tools.md:118]: Replace semicolon with full stop and space

File: content/recipes/interoperability/ontology-operations-tools.md [hunk 2]
```diff
@@ -273,410 +55,138 @@ The table below is an overview of ontology strategies tools identified. Details
...
 ### Ontology mapping
 
 The process of determining correspondences between equivalent concepts in alternative ontologies, and other vocabularies. This may include mapping to convey different levels of granularity.
 
-__Manually curated list of tools__
+__Curated tools__
+
+|Tool|Description|License|Topics|Resource Type|How to use|
+|---|--|--|--|--|--|
+|[OxO](https://www.ebi.ac.uk/spot/oxo/index)|A service for finding mappings (or cross-references) between terms from ontologies, vocabularies and coding standards. |[EMBL-EBI Terms of Use](https://www.ebi.ac.uk/about/terms-of-use/)|Ontology alignment| GUI and API| {ref}`ontology-alignment-oxo` |
```
Comment (EvaMart) [inline content/recipes/interoperability/ontology-operations-tools.md:142]: How-To reference missing?

**Review by EvaMart — APPROVED** [review]

File: content/recipes/interoperability/ontology-operations-tools.md (hunk 1 above)
Comment (FuqiX) [reply content/recipes/interoperability/ontology-operations-tools.md]: thanks for pointing that out. now fixed/

File: content/recipes/interoperability/ontology-operations-tools.md (hunk 2 above)
Comment (FuqiX) [reply content/recipes/interoperability/ontology-operations-tools.md]: the oxo recipe will not be available for now. the ref is removed

**Pull request merged**